src/Iot.Device.Bindings.SkiaSharpAdapter/Directory.Build.props

```diff
@@ -0,0 +1,13 @@
+<Project>
+  <!-- Packaging related properties -->
+  <PropertyGroup>
+    <MajorVersion>2</MajorVersion>
```

Comment:
do we actually care about those versions? Presumably adapter is not getting shipped?

Reply:
wait a sec.. looking at code later seems like this will be a new package, do we actually need it? could this be constructed similarly to other common projects for Iot.Device.Bindings? I might have missed the triage meeting where you discussed this

Reply:
Yes, we want this as a separate package (to reduce dependencies for those who do not need any bitmap functionality or want to use another imaging library). 

I'm not sure how the version for the release is actually computed in our setup though (question for @joperezr )

Reply:
I think we should match, which is the case now (IIRC). But we should (in a separate PR) maybe improve the way the version number is calculated or defined, so we're sure we get the same number everywhere. That is burried somewhere in the arcade repo I guess, and I don't really understand that stuff.